src/components/companies/subscriptions/details.vue

```diff
@@ -114,6 +122,29 @@ export default {
                 name: "plans-modal",
                 scrollable: true
             });
+        },
+
+        cancelSubscription() {
+            if (confirm("Are you sure you want to cancel your subscription?")) {
```

Comment:
Please, no confirm. We have modal components with proper elegant design.

Reply:
@SparoHawk  done, I am using the modal now.